CriticalMass/NetworkActivityIndicatorHelper.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  NetworkActivityIndicatorHelper.swift
+//  CriticalMaps
+//
+//  Created by Leonard Thomas on 5/5/19.
+//
+
+import Foundation
+
+class NetworkActivityIndicatorHelper {
+    private var requestCounter = 0 {
```

Comment:
maybe explicitly `UInt` type here since it should not be a negative value

Reply:
Yeah I also thought about that but was to scared that this might crash at some point in production because it becomes -1 for some wired cases.. We could still do it or maybe at an assert so it only crashes for debug builds

Reply:
💡